build/ansible-service-broker

```diff
@@ -1,5 +1,36 @@
 #!/usr/bin/env bash
 
-BROKER_CONFIG=/etc/ansible-service-broker/broker-config.yaml
+export BROKER_CONFIG=/etc/ansible-service-broker/broker-config.yaml
+
+python - <<END
+import os
+import yaml
+from subprocess import call
```

Comment:
Will `yaml` need to be a requirement in the rpm or is it usually installed with python?